www/src/public/js/controllers/builder/main.js

```diff
@@ -304,6 +313,12 @@
                 baseStats.amulet = encoder.toNumber(listStr.slice(0,1));
             listStr = listStr.substring(1);
 
+            if (statList[0] === '_')
```

Comment:
wrap this in a check for v3 lists

Reply:
Version check included as if statement for hazelnut value.